1694.reformat-phone-number.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * URL of this problem
+ * https://leetcode.com/problems/reformat-phone-number/
+ */
+
+/**
+ * @param {string} number
+ * @return {string}
+ */
+var reformatNumber = function(number) {
+    
+    //Remove hyphens and spaces
+    let integerArr = number.split("").filter((a) => a !== "-" && a !== " ");
+    let arr = [];
+    
+    while (integerArr.length > 4) {
+        arr.push(integerArr.splice(0, 3));
+    }
+    
+    if (integerArr.length === 4) {
+        arr.push(integerArr.splice(0, 2));
+        arr.push(integerArr);
+    } else {
+        arr.push(integerArr);
+    }
+    
+    //Add a hyphen to a number, then turn it into a string
```

Comment:
```suggestion
    // Add hyphens between blocks, then turn it into a string
```
This will make more sense